index.bs

```diff
@@ -1499,7 +1500,10 @@ input parameters:
 - A list of credentials acceptable to the [=[RP]=] (possibly filtered by the client).
 - Extension data created by the client based on the extensions requested by the [=[RP]=].
 
-When this method is invoked, the [=authenticator=] must perform the following procedure:
+The [=authenticator=] must perform the following procedure depending on the PublicKeyCredentialType of the related [=credential=]:
```

Comment:
How does the authenticator find the related credential in order to make this decision? That's not discovered until step 4 of the `"public-key"` steps.